x-pack/metricbeat/module/gcp/timeseries_metadata_collector.go

```diff
@@ -100,6 +100,10 @@ func (s *StackdriverTimeSeriesMetadataCollector) Metadata(ctx context.Context, i
 		// common.Mapstr seems to not work as expected when deleting keys so I have to iterate over all results to add
 		// the ones I want
 		for k, v := range s.timeSeries.Resource.Labels {
+
+			if k == TimeSeriesResponsePathForECSInstanceID {
+				_, _ = ecs.Put(ECSCloudInstanceIDKey, v)
```

Comment:
we are explicitly taking care of only instance_id here because we set availability_zone a few lines above in the same function and also project_id is already being set in cloud.account.id-(this is for all services)